### ISSUE TITLE
typos in shorthand definitions

### ISSUE THREAD

**Issue by mkostal-codasip** [riscv/riscv-p-spec#6]
I believe that in section 2.1
```
u<: Unsinged less than comparison.

u≤: Unsinged less than & equal comparison.

u>: Unsinged greater than comparison
```
should be
```
u<: Unsigned less than comparison.

u≤: Unsigned less than & equal comparison.

u>: Unsigned greater than comparison
```

**Comment by chuanhua** [comment]
I will fix this in the next release. Thanks!

**Comment by chuanhua** [comment]
Fixed in the v0.9.1 release. (976dbe7)
